class/book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Great job so far. what you have done is ok, but please remember the previous reviewer was trying to perfect your work. what he meant was,  attr_accessor creates both the READER & WRITER(getter and setters) methods. attr_reader creates only the reader. attr_writer creates only the writer. so the perfect shortcut will be to use `attr_accessor` to avoid confusion like he precised. `attr_accessor :title, :author, :rentals`.  
- please just make that little change for the book, person and rental class and all is good to go. you can check the lesson shortcuts [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor). 😄

Reply:
Thank you for the explanations